### PR TITLE
Release Kaptain 1.2.0

### PR DESCRIPTION
**Pull request by akirillov**
## Jira Ticket

[D2IQ-77106: Kaptain 1.2.0 release notes and docs update](https://jira.d2iq.com/browse/D2IQ-77106)

**NOTES:**
* The release date in the docs is set to August 20, might require an update if needed
* all docs in `sdk/api` are generated from docstrings in the SDK code
* all docs in `tutorials` are generated from Jupyter Notebooks

## Description of changes being made
Documentation for the new Kaptain release.

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [x] Test all commands and procedures, if applicable.
- [x] Create your PR against `main`. 
- [x] Update all links if you are moving a page.
- [x] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: pages/dkp/kaptain/1.2.0/sdk/private-registries/index.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+navigationTitle: Accessing a Private Docker Registry with Self-Signed Certificates
+title: Configuring Access to a Private Docker Registry with Self-Signed Certificates
+beta: false
+menuWeight: 2
+---
+
+This guide focuses on configuring access to private Docker registry with self-signed certificates and is based on
+`configuration-and-credentials` guide. It focuses on:
+
+* secure automatic mounting of a registry certificate to
```
Comment (cgawron1) [inline pages/dkp/kaptain/1.2.0/sdk/private-registries/index.md:12]: These 2 bullet points don't start with capital letters, is this a problem with consistency?

File: pages/dkp/kaptain/1.2.0/tutorials/fairing/index.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+In this notebook you will go through the steps involved in building a Docker image from a base image (e.g. TensorFlow or PyTorch) and a custom trainer file that defines your machine learning model.
+This image can be used for distributed training or [hyperparameter tuning](../katib).
+You can use the model code you generated with `%%writefile` in [MNIST with TensorFlow tutorial](../training/tensorflow) or [MNIST with PyTorch tutorial](../training/pytorch) or a file of your own choosing.
+
+The Docker image builder process stores (temporary) files in MinIO.
+[MinIO](https://min.io/), an open-source S3-compliant object storage tool, is already included with your Kubeflow installation.
+
+### What You Need
+
+- An executable Python file (e.g. an `mnist.py` trainer - you can extract it from [Pytorch tutorial](../training/pytorch) in case you do not have one handy);
```
Comment (cgawron1) [inline pages/dkp/kaptain/1.2.0/tutorials/fairing/index.md:41]: should that be a period instead of a semicolon at the end?

**Review by cgawron1 — APPROVED** [review]
left comments on possible corrections, but otherwise looks good to me

File: pages/dkp/kaptain/1.2.0/sdk/private-registries/index.md (hunk 1 above)
Comment (akirillov) [reply pages/dkp/kaptain/1.2.0/sdk/private-registries/index.md:12]: My understanding is that if a list is introduced by a complete sentence, each bullet point needs to begin with a capital letter. This is not the case here as it's more like a single sentence split into bullets for readability. But I'd defer this question to @cmastr.

File: pages/dkp/kaptain/1.2.0/tutorials/fairing/index.md (hunk 2 above)
Comment (akirillov) [reply pages/dkp/kaptain/1.2.0/tutorials/fairing/index.md:41]: ouch. yes, likely. I'd suggest skipping this one as (a) this file is generated from Jupyter Notebook and a proper fix will require rebuild and tutorials artifact re-release and (b) we plan to remove fairing tutorials anyway. Maybe we can fix this in the docs PR only.

File: pages/dkp/kaptain/1.2.0/sdk/private-registries/index.md (hunk 1 above)
Comment (cmastr) [reply pages/dkp/kaptain/1.2.0/sdk/private-registries/index.md:12]: consistency is key with bullets, but the rest is up to the writer. This is fine as-is.

**Review by cmastr — APPROVED** [review]

**Pull request merged**